src/components/Popover/types.ts

```diff
@@ -17,9 +18,11 @@ export type PopoverButtonProps = {
     onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
 };
 
+export type VirtualElement = popper.VirtualElement;
+
 export type PopoverExternalProps = {
     /** Allows to use custom anchor and disables `openByHover` and `onClick` for the component */
-    anchorRef?: React.RefObject<HTMLElement>;
+    anchorRef?: React.RefObject<HTMLElement | VirtualElement>;
```

Comment:
Let's refactor to `PopupProps['anchorRef']`

Reply:
ok